src/bindings/c/include/c_api/ie_c_api.h

```diff
@@ -73,1092 +73,1128 @@
+
+            /**
+             * @brief Reads the model from the .xml and .bin files of the IR. Use the ie_network_free() method to free
+             * memory.
+             * @ingroup Core
+             * @param core A pointer to the ie_core_t instance.
+             * @param xml .xml file's path of the IR.
+             * @param weights_file .bin file's path of the IR, if path is empty, will try to read bin file with the same
+             * name as xml and if bin file with the same name was not found, will load IR without weights.
+             * @param network A pointer to the newly created network.
+             * @return Status code of the operation: OK(0) for success.
+             */
+            INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode)
+            ie_core_read_network(ie_core_t* core, const char* xml, const char* weights_file, ie_network_t** network);
+
+            /**
+             * @brief Reads the model from an xml string and a blob of the bin part of the IR. Use the ie_network_free()
+             * method to free memory.
+             * @ingroup Core
+             * @param core A pointer to the ie_core_t instance.
+             * @param xml_content Xml content of the IR.
+             * @param xml_content_size Number of bytes in the xml content of the IR.
+             * @param weight_blob Blob containing the bin part of the IR.
+             * @param network A pointer to the newly created network.
+             * @return Status code of the operation: OK(0) for success.
+             */
+            INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode)
+            ie_core_read_network_from_memory(ie_core_t* core,
+                                             const uint8_t* xml_content,
+                                             size_t xml_content_size,
+                                             const ie_blob_t* weight_blob,
+                                             ie_network_t** network);
+
+            /**
+             * @brief Creates an executable network from a network previously exported to a file. Users can create as
+             * many networks as they need and use them simultaneously (up to the limitation of the hardware resources).
+             * Use the ie_exec_network_free() method to free memory.
+             * @ingroup Core
+             * @param core A pointer to the ie_core_t instance.
+             * @param file_name A path to the location of the exported file.
+             * @param device_name A name of the device to load the network to.
+             * @param config Device configuration.
+             * @param exe_network A pointer to the newly created executable network.
+             * @return Status code of the operation: OK(0) for success.
+             */
+            INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode)
+            ie_core_import_network(ie_core_t* core,
+                                   const char* file_name,
```

Comment:
I suppose we need to revert changes in this file?

Reply:
Yes, reverted.